packages/common/src/store/purchase-content/slice.ts

```diff
@@ -37,7 +37,7 @@ const initialState: PurchaseContentState = {
   extraAmount: undefined,
   extraAmountPreset: undefined,
   error: undefined,
-  stage: PurchaseContentStage.START,
+  stage: PurchaseContentStage.IDLE,
```

Comment:
doesn't this mean it's always IDLE?

Reply:
@rickyrombo Where does it actually get set to `START`? I thought you were going to add that to the beginning of the purchase saga

Reply:
in the reducer for `startPurchaseContentFlow` just a few lines down, just so happens we already are:

https://github.com/AudiusProject/audius-protocol/blob/017ec0b5abb19e72adcaf16d805a89aff146c67c/packages/common/src/store/purchase-content/slice.ts#L62